example_remoteVerify_test.go

```diff
@@ -61,7 +61,7 @@ func Example_remoteVerify() {
 	exampleRepo := registry.NewRepository(remoteRepo)
 
 	// exampleRemoteVerifyOptions is an example of notation.RemoteVerifyOptions.
```

Comment:
It looks like a few of these comments for examples may need the names adjusted to match the changes

Reply:
Nice catch! Updated the code.